### ISSUE TITLE
cleanup scripts

### ISSUE THREAD

**Issue by VeraZab** [plotly/react-plotly.js-editor#209]
there's some unused ones, and some that could be combined

**Comment by nicolaskruchten** [comment]
Is there an advantage to having the CSS scripts as fine-grained as they currently are, or could we just have `make:lib:css` and `make:dist:css`? These scripts don't seem to take very long to run so I feel like we could just have `style.js` and `postcss.js` run in IE and non-IE mode every time. /cc @aulneau

**Comment by nicolaskruchten** [comment]
Also, we currently have a mix of `npm run X` and `yarn X` in the scripts... We should standardize to one or the other. If we go with `yarn X` then we should add Yarn to the devDependencies. Thoughts?

**Comment by aulneau** [comment]
Yeah -- I agree. This is partially left over from when I was making the IE style gen work. I asked @VeraZab about the `dist` functions, and they might not even be used. These tasks were broken out so people could see what was going on, and for the way I had been watching the css / testing if things worked for each. We could totally reduce it into a smaller set of tasks to do both at the same time :)

We had talked about using `npm` vs `yarn`, and I thought we had made the choice to reflect `yarn`, but it might not have been updated yet. cc @bpostlethwaite
